### ISSUE TITLE
Numbers are not numbers

### ISSUE THREAD

**Issue by Jey-Cee** [rg-engineering/ioBroker.daswetter#66]
Numbers seems to be strings.
If type is checked with typoef the result is string.
Tested witht daswetter.0.NextHours.Location_1.Day_1.current.clouds_value and daswetter.0.NextHours.Location_1.Day_1.current.temp_value.

**Comment by rg-engineering** [comment]
@Jey-Cee do you have an idea what's going on here?

here it shows "number"

![grafik](https://user-images.githubusercontent.com/8238668/93118192-3285ba80-f6c0-11ea-9a71-c7c33b8cdd24.png)

but in a script where I use

```
var valFull = getState("daswetter.0.NextHours.Location_1.Day_1.current.clouds_value");

log("object type  " + typeof valFull.val)
```


I get

`object type string`

**Comment by Jey-Cee** [comment]
Seems that your object hast type number but the value of the state is a string. This could happen when you get a string from the API or if you use a function that returns a string instead a number. For example toFixed returns a string. 
So you have to check your value and convert it in an number.

**Comment by rg-engineering** [comment]
@Jey-Cee solved with 3.0.3. can we close that issue?

**Comment by Jey-Cee** [comment]
Yes seems to be ok now, thank you
